exercises/etl/example/etl.d

```diff
@@ -79,6 +78,8 @@ bool aaEqual (const int[dchar] lhs, const int[dchar] rhs)
 
 // throw exception
 {
+	import std.exception : assertThrown;
```

Comment:
The observant reviewer will note that this line has been added to `etl/example` but not to `etl/source`.

well guess what! This test is present in `example` (CI, not served to students) but not in `source` (served to students).

Not a problem that is in the scope of this PR to fix, but clearly a long-standing problem. See https://github.com/exercism/dlang/pull/33 for another case where `example` test were inconsistent with `source` tests.